packit_service/models.py

```diff
@@ -771,3 +781,61 @@ def get_by_pipeline_id(cls, pipeline_id: str) -> Optional["TFTTestRunModel"]:
                 .filter_by(pipeline_id=pipeline_id)
                 .first()
             )
+
+
+class InstallationModel(Base):
+    __tablename__ = "github_installations"
+    id = Column(Integer, primary_key=True, autoincrement=True)
+    # information about account (user/organization) into which the app has been installed
+    account_login = Column(String)
+    account_id = Column(Integer)
+    account_url = Column(String)
+    account_type = Column(String)
+    # information about user who installed the app into 'account'
+    sender_id = Column(Integer)
+    sender_login = Column(String)
+
+    created_at = Column(DateTime, default=datetime.utcnow)
+    repositories = Column(ARRAY(Integer, ForeignKey("git_projects.id")))
```

Comment:
wow, this looks crazy, doing the relation from this side

But I'd say it makes sense. Just that I never did this myself.